docs/conf.py

```diff
@@ -1,18 +1,41 @@
 """Sphinx configuration."""
 import sys
 from pathlib import Path
+import subprocess
 
 sys.path.insert(0, str(Path(__file__).absolute().parent.parent / "src"))
+sys.path.insert(0, str(Path(__file__).absolute().parent.parent / "src" / "py21cmemu"))
 
+master_doc = 'index'
+source_suffix = ".rst"
 
 project = "21cmEMU"
 author = "Daniela Breitman"
 copyright = "2023, Daniela Breitman"
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.autosectionlabel",
+    "sphinx.ext.coverage",
+    "sphinx.ext.doctest",
     "sphinx.ext.napoleon",
     "sphinx_click",
     "myst_parser",
+    "nbsphinx",
+    "IPython.sphinxext.ipython_console_highlighting"
 ]
+out = subprocess.run(["python", "setup.py", "--version"], capture_output=True)
```

Comment:
This won't work because there is no `setup.py` in this package. You should instead just be able to do `import py21cmemu` then set `version = release = py21cmemu.__version__`